.github/workflows/pull-request-check.yml

```
@@ -71,7 +71,39 @@ jobs:
           cd tools/samm-cli
           unset JAVA_TOOL_OPTIONS
           mvn -B clean verify -Dmaven.wagon.httpconnectionManager.ttlSeconds=60
-          mvn -B verify -Pnative -Dmaven.wagon.httpconnectionManager.ttlSeconds=60
+        shell: bash
+
+      - name: Build native image
+        if: matrix.os != 'macos-13'
+        run: |
+          java --version
+          export MAVEN_OPTS="-Xmx4096m"
+          cd tools/samm-cli
+          mvn -B package -Pnative -Dmaven.wagon.httpconnectionManager.ttlSeconds=60
+        shell: bash
+
+      - name: Build native image (Mac)
+        if: matrix.os == 'macos-13'
+        run: |
+          mkdir samm-bundle
```

The samm-bundle directory should be renamed to something unique in order to prevent clashes of the warp-packer cache directories when multiple versions of samm-cli are used, e.g.

```suggestion
          bundle="samm-bundle-DEV-SNAPSHOT-$(date +%s)"
          mkdir ${bundle}
```

(But please verify that `date +%s` works as intended on macOS, at least the GNU version on Linux will print the current Unix timestamp)